tests/usbus_msc/Makefile

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
This will also need a `Makefile.ci`
You can create it with

    make generate-Makefile.ci